pydoctor/sphinx.py

```diff
@@ -15,6 +18,12 @@
 from cachecontrol.caches import FileCache
 from cachecontrol.heuristics import ExpiresAfter
 
+if TYPE_CHECKING:
+    from pydoctor.model import Documentable
+else:
```

Comment:
If you do the `__future__` import you won't need the `else` branch.

Reply:
This construct is here to break the import cycle, since `pydoctor.model` imports `SphinxInventory` from `pydoctor.sphinx`. mypy doesn't care about circular imports, but the Python VM and pytest do.

Reply:
Actually, you are right that the `else` branch could be dropped (while keeping the `if`) when using the `__future__` import. But I'd rather not do that right now to preserve Python 3.6 support.